ale_linters/tex/texlab.vim

```diff
@@ -5,7 +5,9 @@ call ale#Set('tex_texlab_executable', 'texlab')
 call ale#Set('tex_texlab_options', '')
 
 function! ale_linters#tex#texlab#GetProjectRoot(buffer) abort
-    return ''
+    let l:project_root = ale#path#FindNearestDirectory(a:buffer, '.git')
+
+    return !empty(l:project_root) ? fnamemodify(l:project_root, ':h') : fnamemodify(a:buffer, ':h')
```

Comment:
Returning the directory of the current buffer when no .git directory is found may cause ALE to spawn an instance of the language server for every tex file open. See the comment about this: https://github.com/dense-analysis/ale/pull/2501#issuecomment-491993295

Here would be better to return an empty string if no root is found.

Reply:
Good point. I personally didn't mind an LSP server starting for every tex file, but I agree it would not be preferred as the default. I will fix that now, and I'll also try to fix the tests after I understand them more.